### ISSUE TITLE
find a typo in README.md

### ISSUE THREAD

**Issue by evasnowind** [alibaba/Sentinel#1925]
## Issue Description

*bug report*

### Describe what happened (or what feature you want)

in README.md,  I find a typo in this section:
```
|--timestamp-|------date time----|-resource-|p |block|s |e|rt  |occupied
1529998904000|2018-06-26 15:41:44|HelloWorld|20|0    |20|0|0   |0
1529998905000|2018-06-26 15:41:45|HelloWorld|20|5579 |20|0|728 |0
1529998906000|2018-06-26 15:41:46|HelloWorld|20|15698|20|0|0   |0
1529998907000|2018-06-26 15:41:47|HelloWorld|20|19262|20|0|0   |0
1529998908000|2018-06-26 15:41:48|HelloWorld|20|19502|20|0|0   |0
1529998909000|2018-06-26 15:41:49|HelloWorld|20|18386|20|0|0   |0

p stands for incoming request, block for blocked by rules,  success  for success handled by Sentinel, e for exception count, rt for average response time (ms), occupied stands for occupiedPassQps since 1.5.0 which enable us booking more than 1 shot when entering.
```

 `success  for success` should be `s for success`

**Comment by evasnowind** [comment]
I have created an issue to fix this typo

**Comment by evasnowind** [comment]
I have fixed this typo, please review, see https://github.com/alibaba/Sentinel/pull/1926

**Comment by sczyh30** [comment]
Resolved via #1926
